storage/innobase/xtrabackup/test/t/pxb-1914.sh

```diff
@@ -15,12 +15,12 @@ xtrabackup --backup --target-dir=$topdir/bak1 --binlog-info=ON
 
 rm -rf $topdir/bak1
 
-xtrabackup --backup --backup-lock-retry-count=5 --backup-lock-timeout=3 \
+xtrabackup --backup --lock-ddl=false --backup-lock-retry-count=5 --backup-lock-timeout=3 \
 	   --target-dir=$topdir/bak1 --binlog-info=ON
 
 rm -rf $topdir/bak1
 
-xtrabackup --backup --backup-lock-timeout=2 --backup-lock-retry-count=4 \
+xtrabackup --backup --lock-ddl=false --backup-lock-timeout=2 --backup-lock-retry-count=4 \
 	   --no-backup-locks --target-dir=$topdir/bak1
 
 rm -rf $topdir/bak1
```

Comment:
Am in second thoughts about these testcases that use lock-ddl=false

Since we now say, lock-ddl=ON is the recommended and safest way to take backups.
By having these testcases, we are trying to guarantee consistency with lock-ddl=OFF

While it is OK to keep them as is with --lock-ddl=false, it is burden to maintain them.
if these tests fail, we have to put extra effort to fix them.

Reply:
Okay if it fails in future, we can change the behaviour of test case.

Reply:
Ok fine. Please keep an eye on lock-ddl=false failure. We either have to convert to lock-ddl=ON or remove them if they are not applicable with ON.